2D Simulation/game.js

```diff
@@ -409,6 +410,11 @@ function send_user_message_to_server(gameConfig) {
     }
 }
 
+function gesture_is_visible() {
+    gestureElement = document.getElementById('gestureToggle')
```

Comment:
Could you change this to something like `$('#gestureToggle').show()`? There is already a split in the code base between jQuery and vanilla js and we should probably stop contributing to that, myself included.